packages/amplify-appsync-simulator/src/__tests__/server/subscription/websocket-server/server.test.ts

```diff
@@ -76,9 +77,20 @@ describe('WebsocketSubscriptionServer', () => {
     httpServer.listen(serverPort);
     server.start();
   });
+
   afterEach(() => {
-    server.stop();
-    httpServer.close();
+    server?.stop();
+    httpServer?.close();
+  });
+
+  beforeAll(done => {
+    done();
+  });
+
+  afterAll(done => {
+    server?.stop();
```

Comment:
aren't this 2 lines executed in the afterEach section?

Reply:
remove it here